android/filament-android/src/main/cpp/View.cpp

```diff
@@ -483,10 +483,11 @@ Java_com_google_android_filament_View_nIsStencilBufferEnabled(JNIEnv *, jclass,
 extern "C"
 JNIEXPORT void JNICALL
 Java_com_google_android_filament_View_nSetStereoscopicOptions(JNIEnv *, jclass, jlong nativeView,
-        jboolean enabled) {
+        jboolean enabled, jboolean combineMultiviewImages) {
     View* view = (View*) nativeView;
     View::StereoscopicOptions options {
-        .enabled = (bool) enabled
+        .enabled = (bool) enabled,
+        .combineMultiviewImages = (bool) combineMultiviewImages,
```

Comment:
instead of using a boolean here, use the Engine's debugging properties facility:

In `Engine.h` add a field to the `debug` structure:

```
struct {
  (...)
  struct {
     bool combine_multiview_images = false; // or true if you want it to be the defulat
  } stereo;
} debug;
```

Then in `Renderer.cpp` at the top add something like:

```
    debugRegistry.registerProperty("d.stereo.combine_multiview_images",
            &engine.debug.stereo.combine_multiview_images);
```
And finally, you can just test against `engine.debug.stereo.combine_multiview_images`.


An app can set this property like this:
```
auto& debug = engine->getDebugRegistry();
bool* combineMultiViewEnabled = debug.getPropertyAddress<bool>("d.stereo.combine_multiview_images");
*combineMultiViewEnabled = true;
```

Reply:
Great! let me apply this. Thanks!